README.md

```diff
@@ -1 +1,5 @@
-# pyteal-utils
\ No newline at end of file
+# pyteal-utils
+
+*EXPERIMENTAL* *WIP*
```

Comment:
I think it is worthwhile to provide a _brief_ overview of the contents of the repo, even in the initial commit. It's perfectly fine to explain that the code is totally experimental and is expected to change a whole lot, so there are no guarantees regarding the API.